### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard:Focus is not visible on the color radio buttons.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9223]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS2.4.7;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a31101c69aa539f448b9e3ecebc19de5ff
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.4.7

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
4. Navigate to the color radio buttons using tab key.

**Actual:**
On navigating to the radio button using the tab key the focus is not visible as well as using arrow key for navigation.

**Expected:**
The focus should be clearly visible in the form of dotted lines or a dark outline around the buttons.

**User Impact:**
Keyboard users will not know which button they are currently on.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard9223.zip](https://github.com/microsoft/azuredatastudio/files/4322989/A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_Keyboard9223.zip)

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.42.0
![image](https://user-images.githubusercontent.com/85863999/218070971-2215dace-5e98-4a21-9a76-4d9f318f8964.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
